tests/js/spec/components/avatar/avatarList.spec.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import AvatarList from 'app/components/avatar/avatarList';
+
+describe('AvatarList', function() {
+  it('renders with user avatars', function() {
+    const users = [TestStubs.User({id: '1'}), TestStubs.User({id: '2'})];
+
+    const wrapper = mount(<AvatarList users={users} />);
+    expect(wrapper.find('Avatar')).toHaveLength(2);
+    expect(wrapper.find('CollapsedUsers')).toHaveLength(0);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
nit: I would say snapshots here are not that valuable, the two assertions above it are going to be much more important